### PR TITLE
Add Slovak braille specs

### PR DESCRIPTION
**Pull request by pvagner**

### CODE REVIEW

File: slovak/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Slovak Braille Specification
+
+## [Pravidlá písania a používania Braillovho písma
+v Slovenskej republike](pravidla-bp01.pdf)
```
Comment (bertfrees) [inline slovak/README.md]: There seems to be a new line because "písma" and "v". Not sure if that matters.

**Comment by bertfrees** [comment]
You added the file "pravidla-braillovo-pismo-01.txt" but it was not referenced. An oversight?

**Comment by pvagner** [comment]
Hello,
I have fixed the new line in the middle of a heading and referenced the braille text file. Is it done right?

File: slovak/README.md (hunk 1 above)
Comment (pvagner) [reply slovak/README.md]: Fixed

**Review by egli — APPROVED** [review]

**Comment by bertfrees** [comment]
Thanks, merged.

**Pull request merged**